web/src/pages/Route/locales/zh-CN.ts

```diff
@@ -40,6 +40,7 @@ export default {
   'page.route.published': '已发布',
   'page.route.unpublished': '未发布',
   'page.route.onlineDebug': '在线调试',
+  'page.route.service': '服务',
```

Comment:
it would be better to reuse `menu.service` instread of redefine a new key

Reply:
> it would be better to reuse `menu.service` instread of redefine a new key

If menu.serice but the fields are not changed there will be a problem. so I think they should be independent 🤔